pkg/controller/worker.go

```diff
@@ -59,7 +59,7 @@ func processNextWorkItem(shutdownCtx context.Context, controller Interface) bool
 
 	// Run the syncHandler, passing it the namespace/name string of the
 	// Foo resource to be synced.
-	result, err := controller.Reconciler().Reconcile(request)
+	result, err := controller.Reconciler().Reconcile(context.TODO(), request)
```

Comment:
We should pass `shutdownCtx` instead so the reconciling loop can abort sooner if needed in the future.

Reply:
I mimicked this https://github.com/openshift/cluster-resource-override-admission-operator/blob/b1b0669caf0e41710356c411d3e1b0853aab51a2/pkg/controller/worker.go#L62 assuming that there is a reason for not using shuwdownCtx

Reply:
It seems that the reason of why context.TODO is passed to Reconcile function is that it is not used inside of function. Maybe it would be better to pass `shutdownCtx` in case in the future it is used.

Reply:
updated.